pyrobosim/pyrobosim/core/world.py

```diff
@@ -1055,7 +1061,7 @@ def is_connectable(self, start, goal, max_dist=None):
 
         # Build up the array of test X and Y coordinates for sampling between
         # the start and goal points.
-        dist_array = np.arange(0, dist, 0.01)
+        dist_array = np.arange(0, dist, step_dist)
```

Comment:
Removed this hard-coded pose in this PR as well, and made it a parameter of the path planners.